src/pages/state/[state].content.yml

```diff
@@ -49,6 +49,9 @@ videos:
     link: https://youtu.be/TnKo8Q-Hemk?t=3401
     tags:
       - OH
+    description: \[Optional] Lorem ipsum dolor sit amet, consectetur adipiscing
+      elit. Nulla auctor commodo vestibulum. In sollicitudin ultrices blandit.
+      Suspendisse lacus quam, euismod sed facilisis a, venenatis facilisis nisi.
```

Comment:
Should rm before merging, right?

Reply:
Yes, that was my intention all along and I totally wouldn't have forgotten about it... 🤔